### PR TITLE
Document oomRecovery strategy parameters

### PR DESCRIPTION
**Pull request by crezvoy**
Document oomRecovery strategy parameters

# Jira Ticket

[BGD-3130](https://spotinst.atlassian.net/browse/BGD-3130)

# Demo

Screenshot 2023-07-20 at 17 12 51

# Checklist:
- [x] I have performed a self-review of my own code
- [x] I have validated all the requirements in the Jira task were answered

[BGD-3130]: https://spotinst.atlassian.net/browse/BGD-3130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml [hunk 1]
```diff
@@ -1219,6 +1220,31 @@ components:
...
+        strategy:
+          required: true
+          type: string
+          title: oomRecovery
+          const: 'oomRecovery'
+          description: Automatically increase memory if a previous app failed with an OOM error
+        scalingFactor:
+          required: false
+          description: Memory multiplier after an OOM failure. Optional, defaults to 2.0
+          title: title
```
Comment (mathwall) [inline api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml:1240]: I think there is a mistake here

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml [hunk 2]
```diff
@@ -1219,6 +1220,31 @@ components:
...
+          type: number
+        maxMemory:
+          required: false
+          description: Optional, maximum allocatable memory
+          title: title
```
Comment (mathwall) [inline api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml:1245]: here too

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml (hunk 1 above)
Comment (crezvoy) [reply api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml]: 👍

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml (hunk 2 above)
Comment (crezvoy) [reply api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml]: 👍

**Review by mathwall — APPROVED** [review]

**Pull request merged**